platform/switch/os_switch.cpp

```diff
@@ -456,11 +456,10 @@ void OS_Switch::run() {
 			}
 		}
 
+        swkbdInlineUpdate(&inline_keyboard, NULL);
```

Comment:
Why did you move this up?

Reply:
Ah, my bad, that was from when I first tested Stary's suggestion from a few months ago of trying to update the keyboard before the input buffer was flushed

As that was originally theorised to be the problem before you spotted the actual issue - I forgot to revert that, ditto for one of my own printf lines

Reply:
Also note the failed style check when you revert this. It needs to be indented with \t not four spaces.

Reply:
You kinda forgot to fix the rest of the indentation but I got it.